### PR TITLE
Add first GA LabServices API

### PR DESCRIPTION
**Pull request by ianhays**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [X] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [X] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [X] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

PTAL: @sergey-shandar @ravbhatnagar

### CODE REVIEW

**Comment by ianhays** [comment]
PTAL: @sergey-shandar @ravbhatnagar

File: specification/labservices/resource-manager/readme.nodejs.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+## Node.js
+
+These settings apply only when `--nodejs` is specified on the command line.
+Please also specify `--node-sdks-folder=<path to root folder of your azure-sdk-for-node clone>`.
+
+``` yaml $(nodejs)
+nodejs:
+  azure-arm: true
+  package-name: azure-arm-labservices
+  output-folder: $(node-sdks-folder)/lib/services/labServices
```
Comment (daschult) [inline specification/labservices/resource-manager/readme.nodejs.md]:
```suggestion
  output-folder: $(node-sdks-folder)/lib/services/labservicesManagement
```

File: specification/labservices/resource-manager/readme.typescript.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+## TypeScript
+
+These settings apply only when `--typescript` is specified on the command line.
+Please also specify `--typescript-sdks-folder=<path to root folder of your azure-sdk-for-js clone>`.
+
+``` yaml $(typescript)
+typescript:
+  azure-arm: true
+  package-name: "@azure/arm-LabServices"
```
Comment (daschult) [inline specification/labservices/resource-manager/readme.typescript.md]: NPM package names must be all lowercase.

```suggestion
  package-name: "@azure/arm-labservices"
```

File: specification/labservices/resource-manager/readme.typescript.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  output-folder: "$(typescript-sdks-folder)/packages/@azure/arm-LabServices"
```
Comment (daschult) [inline specification/labservices/resource-manager/readme.typescript.md]: This repository uses the package name for its folder structure, so please change this too.
```suggestion
  output-folder: "$(typescript-sdks-folder)/packages/@azure/arm-labservices"
```

**Comment by ravbhatnagar** [comment]
This has been reviewed in the private repo and over skype meetings. There was one major outstanding issue that service team ACKd to fix for the next API version. referenced here - https://github.com/Azure/azure-rest-api-specs/issues/4637

**Comment by hovsepm** [comment]
@ianhays please fix CI build errors.

**Comment by ianhays** [comment]
@hovsepm could you provide some insight on the CI errors? It would appear that they have to do with examples missing, but after adding one the error is the same.

**Comment by daschult** [comment]
Node.js and TypeScript readme configuration files are better now. Thanks!

**Comment by hovsepm** [comment]
@ianhays you need to add example for each operation, not only for `Labs_List`

**Comment by ianhays** [comment]
@hovsepm oof, that's a lot of examples. Is it possible to get an exception of that requirement? Our previous projects were approved without an exception per operation and we'll be GAing this soon.

**Pull request merged**